contribs/src/main/java/com/netflix/conductor/contribs/publisher/RestClient.java

```diff
@@ -0,0 +1,58 @@
+package com.netflix.conductor.contribs.publisher;
+
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+import org.apache.http.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RestClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RestClient.class);
+    //private static final String URL = "http://bullwinkle.default:7979/v1/workflow";
+    private static final String URL = "http://bullwinkle.default.svc.cluster.local:7979/v1";
+    private static final String HEADER_DOMAIN_GROUP = "X-Starship-DomainGroup";
+    private static final String HEADER_ACCOUNT_COOKIE = "x-barracuda-account";
+    private static final String HEADER_PREFER = "Prefer";
+    private static final String HEADER_PREFER_VALUE = "respond-async";
+    private Client client = Client.create();
+    // default timeout value for all requests
```

Comment:
Is this needed?

Reply:
This we added to timeout in 10sec in case of no response from bullwinkle